oidc-client-app-samples/android-client-app-sample/settings.gradle

```diff
@@ -0,0 +1 @@
+include ':app'
```

Comment:
Missing license header.

Reply:
Fixed in PR: https://github.com/wso2/samples-is/pull/179